examples/multibot.py

```diff
@@ -40,7 +40,7 @@ def is_bot_token(value: str) -> Union[bool, Dict[str, Any]]:
 
 
 @main_router.message(Command(commands=["add"], command_magic=F.args.func(is_bot_token)))
-async def command_add_bot(message: Message, command: CommandObject, bot: Bot):
+async def command_add_bot(message: Message, command: CommandObject, bot: Bot) -> Any:
```

Comment:
Why in some cases you change `Any -> None`, is some other cases change `None -> Any`?

Reply:
`Any -> None` because there is no any `return`
https://github.com/aiogram/aiogram/blob/ad4ccac8f7db470d6eb76b676fb4b7dbaab67f9b/examples/echo_bot.py#L25-L37

`None -> Any` because there is optional SendMessage `return`. May be `Optional[TelegramMethod]` would be more correct.
https://github.com/aiogram/aiogram/blob/ad4ccac8f7db470d6eb76b676fb4b7dbaab67f9b/examples/multibot.py#L42-L51